recipes/scikit-sparse/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.3" %}
+
+package:
+  name: scikit-sparse
+  version: {{ version }}
+
+source:
+  fn: scikit-sparse--{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-sparse/scikit-sparse-{{ version }}.tar.gz
+  sha256: 2456be16e65bb8126ea7b9a3b175282b396d07ebeb0b4c5ac85d707bfc977008
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - numpy x.x
+    - suitesparse
+    - cython
+    - setuptools
+  run:
+    - python
+    - numpy x.x
+    - scipy
+    - suitesparse
+    - setuptools
```

Comment:
I see. I think this issue was encountered when `bob.ip.flandmark` was around. So assumed it was related to that.

Honestly, I can see this both ways. Though I think you are right to move it to `test` requirements. We did the same thing for `nose` with `numpy`.